src/visualizers/widgets/OperationCodeEditor/OperationCodeEditorWidget.js

```diff
@@ -18,10 +18,10 @@ define([
         TextEditorWidget.call(this, logger, container);
         this.lineOffset = 0;
         // Add the shift-enter command
-        this.editor.addCommand(
-            monaco.KeyMod.Shift | monaco.KeyCode.Enter,
-            this.executeOrStopJob
-        );
+        // this.editor.addCommand(
```

Comment:
Why did you remove this?

Reply:
There has been an additional check in the `addCommand` function where it checks if the handler is undefined (on update to new version of the monaco-editor). Since the widget gets initialized before control, I have changed it to be a function in the widget itself. Commit Reference [52272ed](https://github.com/deepforge-dev/deepforge/pull/2063/commits/52272edc05ecc0a289c38bd192651995e15e63dc).